frontend/discourse/app/components/search-bulk-select-dropdown.gjs

```diff
@@ -0,0 +1,104 @@
+import Component from "@glimmer/component";
+import { array } from "@ember/helper";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import BulkSelectTopicsDropdown from "discourse/components/bulk-select-topics-dropdown";
+import Post from "discourse/models/post";
+import { i18n } from "discourse-i18n";
+
+export default class SearchBulkSelectDropdown extends Component {
+  @service dialog;
+
+  get topics() {
+    const topics = new Map();
+    (this.args.bulkSelectHelper.selected || []).forEach((post) => {
+      if (post.topic) {
+        topics.set(post.topic.id, post.topic);
+      }
+    });
+    return Array.from(topics.values());
+  }
+
+  get topicBulkSelectHelper() {
+    const topics = this.topics;
+    return {
+      selected: topics,
+      selectedIds: topics.map((t) => t.id),
+      selectedCategoryIds: [
+        ...new Set(topics.map((t) => t.category_id).filter(Boolean)),
+      ],
+      dismissRead: (operationType, options) =>
+        this.args.bulkSelectHelper.dismissRead(operationType, options, topics),
```

Comment:
I'm not sure I understand why we need to accept `options` as the arguments when `dismissRead` is only going to be called with `operationType`: 

https://github.com/discourse/discourse/blob/a65e22476bc6d02a107d3fca2ae5ea9f05a9f4bc/frontend/discourse/app/components/bulk-select-topics-dropdown.gjs#L295

Also, passing `topics` as an argument to `this.args.bulkSelectHelper.dismissRead` doesn't seem to do anything: https://github.com/discourse/discourse/blob/a65e22476bc6d02a107d3fca2ae5ea9f05a9f4bc/frontend/discourse/app/lib/bulk-select-helper.js#L47